backend/src/routers/user.ts

```diff
@@ -118,4 +124,120 @@ userRouter.get(
     })
 );
 
+userRouter.get(
+    '/projects/',
+    expressAsyncHandler(async (req, res) => {
+        try {
+            const payload = await readToken(req);
+            if (payload === null) res.status(401).end();
+            else {
+                const response = getProjects(payload.userID);
+                res.json(response).status(200);
+            }
```

Comment:
Add a return here.

Reply:
Instead of adding a return I would have preferred moving res.status(400).end(); inside the catch statement.
This is a personal preference and not a requirement.